<launch>
  <node name="rosserial_python" pkg="rosserial_python" type="serial_node.py" respawn="true" args="/dev/ttyACM0" output="screen"/>
  <node name="encoder_filter" pkg="motor_control" type="encoder_filter"/>
  <node name="motor_control" pkg="motor_control" type="motor_control"/>
  <node name="odometry_data" pkg="odometry_data" type="odometry_data"/>
  <node name="ir_obstacle_detection" pkg="ir_obstacle_detection" type="ir_obstacle_detection"/>
  <node name="global_position_estimation" pkg="global_position_estimation" type="global_position_estimation"/>
  <node name="decision_node" pkg="decision_node" type="decision_node"/>
  <node name="map_creation" pkg="map_creation" type="map_creation"/>
</launch>
